README.md

```diff
@@ -93,6 +93,7 @@ The default configuration is currently set to:
 
 ```json
 {
+  "automergeSchedule": ["after 9am and before 4pm every weekday"],
```

Comment:
I don't how everyone else are, but 80% of partner integration members is asleep before 10 :), I would put it after 1 or even 2 pm.

This is more controversial I suppose – as a default, I would also put it as once a week, and maybe add exception for `firestore-schemas` to update it more frequently.

Reply:
1. I get your point, I think when we look at "default" business hours I think of 09:00-17:00, so I think we use that timeframe minus some buffer on both ends. (I'll update the schedule to be from 10:00-16:00.) And for your specific case I'd prefer an override in the repositories your team maintains tbh.
1. To me the main idea of the `automergeSchedule` is that those are only happening while we are at work and not bothering on call engineers with runtime issues due to automatically merged dependency updates like we've seen here https://www.notion.so/ridedott/Dependency-update-causing-runtime-errors-20cc8552b6bc494d962b0cdd8402d220.  In general I'd abstain from having exceptions to this, even though I agree with those `firestore-schema` PRs usually being safe in this regard.

Reply:
@markelog can you have a look at this again?

Reply:
@markelog pinging you here again, thanks for having another look!

Reply:
Sorry for delay, a bit of vacation and after vacation hassle got me busy